getting-started/windows-calico/kubernetes/standard.md

```diff
@@ -250,18 +250,8 @@ PS C:\> Start-Service RemoteAccess
 4. Verify that the {{site.prodname}} services are running.
 
    ```powershell
-   PS C:\> Get-Service -Name CalicoNode
-
-   Status   Name               DisplayName
-   ------   ----               -----------
-   Running  CalicoNode         Calico Windows Startup
-
-
-   PS C:\> Get-Service -Name CalicoFelix
-
-   Status   Name               DisplayName
-   ------   ----               -----------
-   Running  CalicoFelix        Calico Windows Agent
```

Comment:
I feel like the output isn't necessary but we can keep this if you think it's useful.